azure/azure.go

```diff
@@ -371,14 +371,20 @@ func (a *azureOps) Create(
 	options map[string]string,
 ) (interface{}, error) {
 	d, ok := template.(*compute.Disk)
-	if !ok || d.DiskProperties == nil || d.DiskProperties.DiskSizeGB == nil {
+	if !ok || d.DiskProperties == nil {
```

Comment:
According to the documentation of Azure -  If creationData.createOption is Empty, this field is mandatory and it indicates the size of the disk to create.

Also, this parameter caused panic before - https://portworx.atlassian.net/browse/PWX-29069